frontend/packages/kubevirt-plugin/src/hooks/use-local-storage.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+export const useLocalStorage = (key: string): [string, React.Dispatch<string>] => {
+  const [value, setValue] = React.useState(window.localStorage.getItem(key));
```

Comment:
just an idea, I guess this is ok approach, but shouldn't we rather try to store an object under one key like `kubevirt-plugin` to prevent the store polution?